src/field/email.js

```diff
@@ -1,18 +1,18 @@
-import trim from 'trim';
 import { setField } from './index';
-import { endsWith } from '../utils/string_utils';
 import { isHRDEmailValid } from '../connection/enterprise';
 import * as i18n from '../i18n';
 
-const regExp = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
-
 export function validateEmail(str) {
   return isEmail(str);
 }
 
 export function isEmail(str) {
-  const result = regExp.exec(trim(str.toLowerCase()));
```

Comment:
I think you should keep the "trim" (removes spaces on edges of the string) to conserve the old behavior.

Reply:
fixed